src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/vilkårsvurdering/preutfylling/PreutfyllBosattIRiketServiceTest.kt

```diff
@@ -475,4 +476,34 @@ class PreutfyllBosattIRiketServiceTest {
                 "- Bosatt i Norge siden fødsel.",
         )
     }
+
+    @Test
+    fun `skal preutfylle bosatt i riket vilkår i EØS saker`() {
+        // Arrange
+        val behandling = lagBehandling(behandlingKategori = BehandlingKategori.EØS)
+        val persongrunnlag = lagTestPersonopplysningGrunnlag(behandling.id, søkerPersonIdent = randomFnr(), barnasIdenter = listOf(randomFnr()))
+        val vilkårsvurdering = lagVilkårsvurdering(persongrunnlagForBehandling = persongrunnlag, behandling = behandling)
+        val personResultat = lagPersonResultat(vilkårsvurdering = vilkårsvurdering)
+
+        every { pdlRestClient.hentStatsborgerskap(personResultat.aktør, historikk = true) } returns
+            listOf(
+                Statsborgerskap("SWE", LocalDate.now().minusYears(10), null, null),
+            )
+
+        every { pdlRestClient.hentBostedsadresserForPerson(any()) } returns
+            listOf(
+                Bostedsadresse(
+                    gyldigFraOgMed = LocalDate.now().minusYears(1),
+                    vegadresse = lagVegadresse(12345L),
+                ),
+            )
+
+        // Act
+        val vilkårResultat = preutfyllBosattIRiketService.genererBosattIRiketVilkårResultat(personResultat = personResultat)
+
+        // Assert
+        assertThat(vilkårResultat).hasSize(1)
+        assertThat(vilkårResultat.find { it.vilkårType == Vilkår.BOSATT_I_RIKET }).isNotNull
+        assertThat(vilkårResultat.find { it.resultat == Resultat.OPPFYLT }).isNotNull
```

Comment:
Gir det mer mening å skrive det slik? 

```suggestion
        assertThat(vilkårResultat).allSatisfy {
            assertThat(it.vilkårType).isEqualTo(Vilkår.BOSATT_I_RIKET)
            assertThat(it.resultat).isEqualTo(Resultat.OPPFYLT)
        }
```

Evt. slik?

```suggestion
        assertThat(vilkårResultat.first().vilkårType).isEqualTo(Vilkår.BOSATT_I_RIKET)
        assertThat(vilkårResultat.first().resultat).isEqualTo(Resultat.OPPFYLT)
```